MHW_ModManager/Lang/zh-CN/From.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I think we should stick to the form's resource files for form UI translations, Since there is no easy way to automatically update these key values whenever we change the name of a form control.